### ISSUE TITLE
No neural network module

### ISSUE THREAD

**Issue by wilhelmagren** [neurocode-ai/leaf#18]
Start implementing leaf.nn, we want at least modules:
- Linear
- Conv2d
- Pool2d
- BatchNorm2d
- Activation functions
- lstm (?)

**Comment by wilhelmagren** [comment]
This is done. Open issue for specific module implementations.
